making-programming-videos/README.md

```diff
@@ -0,0 +1,32 @@
+# How to Make Engaging Programming Videos
+
+This repository provides the VSCode settings file that is mentioned in the Real Python [How to Make Engaging Programming Videos](https://realpython.com/making-programming-videos/) tutorial.
```

Comment:
@acsany VS Code is spelled with a space in between <https://code.visualstudio.com/docs>

Just mentioning it because I've seen you use it consistently spaceless in the tutorial and here.